### PR TITLE
add Arweave Dev.md

### PR DESCRIPTION
**Pull request by jhfnetboy**
day day up~

### CODE REVIEW

**Review by qizhou — APPROVED** [review]

File: basic/43-Arweave/Dev.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+## Dev preparation
```
Comment (luckyyang) [inline basic/43-Arweave/Dev.md:1]: Could you add a description to illustrate what's the purpose of this doc? What the reader will get after read this doc?

File: basic/43-Arweave/Dev.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+### Create a AR wallet
++ get Chrome extension [here](https://chrome.google.com/webstore/detail/arweave/iplppiggblloelhoglpmkmbinggcaaoc)
++ [More help](https://docs.arweave.org/info/wallets/arweave-web-extension-wallet)
++ If you forget the encryption passphrase, just re-import the downloaded json file(It is very important to keep security)
+## Http
+### Config
++ The default port is 1984.
++  http://159.65.213.43:1984/info， 或者 https://arweave.net/info.
```
Comment (luckyyang) [inline basic/43-Arweave/Dev.md:10]: `或者` should be `or` ?

File: basic/43-Arweave/Dev.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+```
+# arkb balance
+pEbU_SLfRzEseum0_hMB1Ie-hqvpeHWypRhZiPoioDI has a balance of 10.113659492352 AR
+```
++ [More](https://docs.arweave.org/developers/tools/textury-arkb)
+
+## PermonWeb
++ Permaweb apps are built using normal web technologies — HTML, CSS, and Javascript — but are deployed to Arweave’s on-chain storage system, making them permanent and available in a fast, decentralised manner. 
++ You can deploy a permaweb app in 2 minutes or less with Arweave Deploy!
++ [More](https://docs.arweave.org/developers/hackathon)
```
Comment (luckyyang) [inline basic/43-Arweave/Dev.md:65]: Change to this?
[Docs for hackathon](https://docs.arweave.org/developers/hackathon)

File: basic/43-Arweave/Dev.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
++ https://arweave.net/tx/{id}/status
++ https://arweave.net/tx/{id}/{field}
++ http://arweave.net/{id}
++ http://arweave.net:1984/tx/{id}/data.{extension}
+
+### More
++ [http API](https://docs.arweave.org/developers/server/http-api)
+
+## Upload tool arkb
++
```
Comment (luckyyang) [inline basic/43-Arweave/Dev.md:37]: Seems this `+` symbol is not useful, could remove them? （There are 3 more below)

File: basic/43-Arweave/Dev.md (hunk 2 above)
Comment (jhfnetboy) [reply basic/43-Arweave/Dev.md:10]: haha, ok!

File: basic/43-Arweave/Dev.md (hunk 1 above)
Comment (jhfnetboy) [reply basic/43-Arweave/Dev.md:1]: basic information for preparing the developing

File: basic/43-Arweave/Dev.md (hunk 3 above)
Comment (jhfnetboy) [reply basic/43-Arweave/Dev.md]: ok，good idea

File: basic/43-Arweave/Dev.md (hunk 4 above)
Comment (jhfnetboy) [reply basic/43-Arweave/Dev.md:38]: Here is ok

**Review by liberhe — APPROVED** [review]

File: package.json [hunk 5]
```diff
@@ -15,6 +15,7 @@
   ],
   "license": "ISC",
   "dependencies": {
+    "arweave": "^1.10.22",
```
Comment (maroonstar) [inline package.json:18]: why add global lib?

**Review by maroonstar — CHANGES_REQUESTED** [review]

**Review by maroonstar — APPROVED** [review]

**Review by sansx — APPROVED** [review]

**Pull request merged**